client/stylesheets/_toCat.scss

```diff
@@ -2116,8 +2139,9 @@ i:hover {
 
 .contactSend {
     font-family: 'Roboto' !important;
-    margin: 0 29.1% 100px 35%!important;
-    text-align: center !important;
+    margin: 0 0 100px 0!important;
+    text-align: right !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces